CMakeLists.txt

```diff
@@ -128,16 +111,5 @@ INCLUDE_DIRECTORIES( . ./source/Digitisers/include ./source/Refitting/include ./
 # add library
 FILE( GLOB_RECURSE library_sources "source/*.cc" )
 
-# needed for adding header files to xcode project
-IF(CMAKE_GENERATOR MATCHES "Xcode")
-  FILE( GLOB_RECURSE library_headers "*.h" )
-  ADD_SHARED_LIBRARY( ${PROJECT_NAME} ${library_sources} ${library_headers})
-ELSE()
-  ADD_SHARED_LIBRARY( ${PROJECT_NAME} ${library_sources} )
-ENDIF()  
-
-INSTALL_SHARED_LIBRARY( ${PROJECT_NAME} DESTINATION lib )
-
-# display some variables and write them to cache
-DISPLAY_STD_VARIABLES()
+ADD_LIBRARY( ${PROJECT_NAME} SHARED ${library_sources} )
```

Comment:
No strong opinion on removing this. It's still present in iLCSoft/MarlinTrkProcessors. However what definitely has to stay is the installation of the library that we build, as otherwise we will not be able to use the processors when we use this into a software stack.